events/janus_mqttevh.c

```diff
@@ -369,7 +384,7 @@ static void janus_mqttevh_client_reconnect_success(void *context, MQTTAsync_succ
 	}
 	json_t *info = json_object();
 	json_object_set_new(info, "event", json_string("connected"));
```

Comment:
Shouldn't the two lines above be removed, since we're not using this statically? (`janus_mqttevh_send_message` is commented out)

Reply:
Yep, it was there and I decided to not touch this part for now. I'll be happy to remove dead code. :)